apps/communications/contacts/js/views/settings.js

```diff
@@ -713,6 +716,7 @@ contacts.Settings = (function() {
     newOrderByLastName = !orderCheckBox.checked;
     utils.cookie.update({order: newOrderByLastName});
     updateOrderingUI();
+    Cache.evict();
```

Comment:
We reload the list when closing the settings page if we changed the order. That causes the cache rebuilt.